pkg/apim/APIMUtility.go

```diff
@@ -16,6 +18,18 @@ type Utils interface {
 	IsPayloadJSON() bool
 }
 
+//OdataUtils for apim
+type OdataUtils interface {
+	MakeOdataQuery() (string, error)
+}
+
+//OdataParameters struct
+type OdataParameters struct {
+	Filter, Search          string
+	Top, Skip               int
+	Orderby, Select, Expand string
```

Comment:
Don't we need qs annotations here?

Reply:
no, if you see its usage explained here https://pkg.go.dev/github.com/pasztorpisti/qs